core/common/src/main/java/alluxio/master/PollingMasterInquireClient.java

```diff
@@ -112,9 +113,12 @@ private void pingMetaService(InetSocketAddress address) throws AlluxioStatusExce
         .build();
     ServiceVersionClientServiceGrpc.ServiceVersionClientServiceBlockingStub versionClient =
         ServiceVersionClientServiceGrpc.newBlockingStub(channel);
+    ServiceType serviceType
+        = address.getPort() == mConfiguration.getInt(PropertyKey.JOB_MASTER_RPC_PORT)
+        ? ServiceType.JOB_MASTER_CLIENT_SERVICE : ServiceType.META_MASTER_CLIENT_SERVICE;
```

Comment:
Will this work with security? Since the channel will be unauthenticated? @ggezer

Reply:
ServiceVersionClientServiceGrpc is an unauthenticated service for pinging purposes, so it should work.

Reply:
Thanks!